### DISCUSSION TITLE
Persona - Can't remove background color + can't change alignment

### DISCUSSION THREAD

**Discussion by e-gallis** [microsoft/powercat-creator-kit#334] [Q&A] [answered]
Hi @mikefactorial, @BMeganPP, @denise-msft,

I was planning to use the Persona component in an application but unfortunately I discovered that (see capture below):
- the white background cannot be removed 
- we can't set the alignment to show the persona details on the left hand side of the picture

![image](https://github.com/microsoft/powercat-creator-kit/assets/32480338/37c1476e-31b1-40ce-9eba-f4d168914c42)

Is there any chance that you could enhance this component to answer those needs?

Thanks

**Comment by Ramakrishnan24689** [comment, marked as answer]
Hi @e-gallis,

Thanks for reaching out.

You can control the white background color of Creator kit component using the `Theme` Property as pass specific `ThemeJSON` based on your color schema preference.

Kindly refer the theming section of this documentation - https://learn.microsoft.com/en-us/power-platform/guidance/creator-kit/theme

Best/Easiest way to generate the `ThemeJSON` is to use[ Fluent Theme Designer App](https://learn.microsoft.com/en-us/power-platform/guidance/creator-kit/creator-kit-explained#fluent-theme-designer-app) which comes as part of [Creator kit reference solution](https://learn.microsoft.com/en-us/power-platform/guidance/creator-kit/setup#step-2-install-the-reference-solutions-optional).

<img width="129" alt="image" src="https://github.com/microsoft/powercat-creator-kit/assets/11135056/dd34ae75-fe4e-4987-9c6c-3d2f682f6c33">
